src/strings/en-gb.json

```diff
@@ -1769,6 +1769,8 @@
     "ListView": "List View",
     "LabelBackdropScreensaverInterval": "Backdrop Screensaver Interval",
     "LabelBackdropScreensaverIntervalHelp": "The time in seconds between different backdrops when using the backdrop screensaver.",
+    "LabelScreensaverTimeHelp": "The amount of time in seconds of inactivity required to start the screensaver.",
```

Comment:
Don't add strings to the languages other than `en-us`. Only Weblate should be used to work with them.